### PR TITLE
R doc 1710 backup config

### PR DESCRIPTION
**Pull request by morhilai**
Added new config options to Backup Options page, plus some formatting and linking related articles

### CODE REVIEW

File: Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Configuration: Backup
+
+---
+
+{NOTE: }
+
+* Configuration options for backups (both on premise and RavenDB Cloud)  
+
+* In this page:  
+    * [Server.LocalRootPath](../../server/configuration/backup-configuration#server.localrootpath)
```
Comment (ppekrol) [inline Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown:10]: Should be `Backup.` not `Server.` (all items here)

File: Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    * [Server.AllowedDestinations](../../server/configuration/backup-configuration#server.alloweddestinations)  
+    * [Server.AllowedAwsRegions](../../server/configuration/backup-configuration#server.allowedawsregions)  
+    * [Server.MaxNumberOfConcurrentBackups](../../server/configuration/backup-configuration#server.maxnumberofconcurrentbackups)  
+    * [Server.LowMemoryBackupDelayInMin](../../server/configuration/backup-configuration#server.lowmemorybackupdelayinmin)  
+
+{NOTE/}
+
+---
+
+{PANEL:Server.LocalRootPath}
```
Comment (ppekrol) [inline Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown:20]: `Backup.`

File: Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown]: Fixed
What a strange mistake, I wonder how I did that...

File: Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown (hunk 2 above)
Comment (morhilai) [reply Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown]: Fixed

File: Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown]: The 4.0 version of the page is all `Server.` - should I fix that as well?

Comment (ppekrol) [reply Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown]: Leave it

**Pull request merged**